src/layout/KeyValue.tsx

```diff
@@ -9,33 +9,53 @@ export interface Props { state: LiveSplit.KeyValueComponentStateJson }
 
 export default class KeyValue extends React.Component<Props> {
     public render() {
+        const keyCell = <td
+            className="key-value-key"
+            style={{
+                color: map(this.props.state.key_color, colorToCss),
+            }}
+        >
+            {this.props.state.key}
+        </td>;
+
+        const valueCell = <td
+            className="key-value-value time"
+            style={{
+                color: map(this.props.state.value_color, colorToCss),
+            }}
+        >
+            {this.props.state.value}
+        </td>;
+
+        let keyValueRows;
+        if (this.props.state.display_two_rows) {
+            keyValueRows = <>
+                <tr>
+                    {keyCell}
+                </tr>
+                <tr>
+                    {valueCell}
+                </tr>
+            </>;
+        } else {
+            keyValueRows = <>
+                <tr>
+                    {keyCell}
+                    {valueCell}
+                </tr>
+            </>;
```

Comment:
I didn't know this is a thing in React. Can we use this in other places as well?

Reply:
I think it's equivalent to an array of elements, so we could use it whenever we do that. However, it actually shouldn't be used in the specific line that you commented on because it's surrounding a single `tr`, and is therefore useless.